app/pulp/app/serializers/consumer.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+
+from pulp.app import models
+from pulp.app.serializers import NotesKeyValueRelatedField, ModelSerializer
+
+
+class ConsumerSerializer(ModelSerializer):
+    name = serializers.CharField(
+        help_text="The consumer common name."
+    )
+
+    description = serializers.CharField(
+        help_text="An optional description.",
+        required=False
+    )
+
+    notes = NotesKeyValueRelatedField()
+
+    publishers = serializers.HyperlinkedRelatedField(
+        many=True,
+        read_only=True,
+        view_name='publishers-detail'
+    )
```

Comment:
If we're going to be defining this a lot, it might make sense to create a PublisherRelatedField in serializers.fields, but I'm happy for that to be an optimization that happens later.